src/lib/components/projects/Collaborators.svelte

```diff
@@ -74,20 +114,38 @@
   {:else}
     <Empty>
       {$_("projects.collaborators.empty")}
-      {#if $me}
-        <Action on:click={() => (edit = true)}>
+      {#if project.add_remove_access}
+        <Action title={actions.invite} on:click={() => (show = "invite")}>
           {$_("projects.collaborators.add")}
         </Action>
       {/if}
     </Empty>
   {/each}
+
+  {#if users.length > 0 && project.add_remove_access}
```

Comment:
Move to top of list, so that if a project has lots of members it's easy to find this action.

Reply:
I was back and forth on this, but put it at the bottom so you need to at least skim the list of people before adding someone who might already be there, but I'm not set on that.